File: go/242.go
func isAnagram(s string, t string) bool {
    size1, size2 := len(s), len(t)
    if size1 != size2 {
        return false
    }
    dict := make([]byte, 26)
    for _, c := range s {
        dict[c-'a']++
    }
    for _, c := range t {
        dict[c-'a']--
    }
    for _, c := range dict {
        if c != 0 {
            return false
        }
    }
    return true
}

File: cpp/242.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int size1 = s.size(), size2 = t.size();
        if (size1 != size2)
            return false;
        int dict[26] = {0};
        for (auto c : s)
            ++dict[c-'a'];
        for (auto c : t)
            --dict[c-'a'];
        for (int i = 0; i < 26; ++i)
            if (dict[i] != 0)
                return false;
        return true;
    }
};
